lib/mix/test/mix/task_test.exs

```diff
@@ -197,11 +197,11 @@ defmodule Mix.TaskTest do
     assert Mix.Task.preferred_cli_env(:no_task) == nil
   end
 
-  test "preferred_cli_env/1 returns nil when task does not have `preferred_cli_env` attribute" do
+  test "preferred_cli_env/1 returns nil when task does not have :preferred_cli_env attribute" do
```

Comment:
It should be `@preferred_cli_env`
https://github.com/elixir-lang/elixir/blob/6757fbd9c68d18f627f53d0c631cdacb36905506/lib/mix/lib/mix/task.ex#L29

Reply:
Right. I had doubts about this one. Thanks!